content/docs/install.md

```diff
@@ -1,86 +1,94 @@
 ---
 title: Getting Started
-summary: Getting Started with Odin. Downloading, installing, and getting your first program to compile and run.
+summary: Getting Odin and running your first program.
 weight: 1
 ---
 
-Odin is dead-simple to get started with!
+## Getting Odin
+There are two options for getting Odin:
+- A) Download a pre-built Odin binary.
+- B) Clone the Odin repository and build it yourself (**recommended** to make [updating Odin](#update-odin) easier).
 
-<a href="https://github.com/odin-lang/Odin/releases" class="btn btn-outline-primary">Latest Release</a>
-<a href="/docs/nightly" class="btn btn-outline-primary">Latest Nightly Builds</a>
+## Download Odin
+<a href="https://github.com/odin-lang/Odin/releases/latest" class="btn btn-outline-primary">Latest Release Build</a>
+<a href="https://odin-lang.org/docs/nightly/#nightly-builds" class="btn btn-outline-primary">Latest Nightly Build</a>
 
+- Choose either a release build or a nightly build.
+- Download the build that is specific to your platform.
+- Unzip it and move the directory to your desired location.
+- [Add the Odin directory to your `$PATH`.](#add-odin-to-your-path)
+- Use `odin --help` to learn how to run your first Odin program.
 
-## Clone or download Odin binaries
-You can either:
+## Clone and build Odin
```

Comment:
It needs to be made clear that you can either Download Odin **or** Clone and build Odin, as how it currently laid out, it looks you need to do both.

Reply:
I updated the wording, section names, and structure to make that clear.